src/plugins/console/public/application/containers/main/main.tsx

```diff
@@ -71,6 +74,14 @@ export function Main({ dataSourceId }: MainProps) {
   const [showingHistory, setShowHistory] = useState(false);
   const [showSettings, setShowSettings] = useState(false);
   const [showHelp, setShowHelp] = useState(false);
+  const [showImportFlyout, setShowImportFlyout] = useState(false);
+
+  const onExport = async () => {
+    const results = await objectStorageClient.text.findAll();
+    const senseData = results.sort((a, b) => a.createdAt - b.createdAt)[0];
+    const blob = new Blob([JSON.stringify(senseData || {})], { type: 'application/json' });
+    saveAs(blob, 'sense.json');
```

Comment:
Why is this named `sense.json`?

Reply:
At the time, no other name really came to mind, and I had a good experience with the 'Sense' Chrome extension for Elasticsearch queries, so I went with 'sense.json'! 😄

>